astroquery/neodys/core.py

```diff
@@ -0,0 +1,107 @@
+import requests
+from . import conf
+from ..query import BaseQuery
+
+
+__all__ = ['NEODyS', 'NEODySClass']
+
+
+class NEODySClass(BaseQuery):
+
+    NEODYS_URL = conf.server
+    TIMEOUT = conf.timeout
+
+    def __init__(self):
+        super(NEODySClass, self).__init__()
+
+    def query_object(self, object_id, orbital_element_type="eq", epoch_near_present=0):
```

Comment:
Maybe make the optional ones kwarg only.
```suggestion
    def query_object(self, object_id, *, orbital_element_type="eq", epoch_near_present=0):
```